docs/src/components/productMaterial/MaterialDesignKits.tsx

```diff
@@ -1,75 +1,82 @@
-  };
+interface MaterialDesignKitsProps {
+  gradient?: boolean;
+}
+
+export default function MaterialDesignKits({ gradient }: MaterialDesignKitsProps) {
+  const [customized, setCustomized] = React.useState(true);
+
   return (
-    <Section cozy>
+    <Section bg={gradient ? 'gradient' : 'white'} cozy>
       <Grid container spacing={2} alignItems="center">
         <Grid item md={6} sx={{ minWidth: 0 }}>
-          <Box sx={{ maxWidth: 500 }}>
-            <SectionHeadline
-              overline="Design kits"
-              title={
-                <Typography variant="h2">
-                  Enhance your <GradientText>design workflow</GradientText>
-                </Typography>
-              }
-              description="The Design kits contain many of the Material UI components with states, variations, colors, typography, and icons. We frequently update it to sync with the most up-to-date release."
-            />
-          </Box>
-          <Group desktopColumns={2} sx={{ mt: 4 }}>
-            {DEMOS.map((name) => (
-              <Highlighter key={name} selected={name === demo} onClick={() => setDemo(name)}>
-                <Item
-                  icon={React.cloneElement(icons[name], name === demo ? { color: 'primary' } : {})}
-                  title={name}
-                />
-              </Highlighter>
-            ))}
-            <More
-              component={Link}
-              href="https://mui.com/store/?utm_source=marketing&utm_medium=referral&utm_campaign=design-cta3#design"
-              noLinkStyle
-            />
+          <SectionHeadline
+            overline="Design tools"
+            title={
+              <Typography variant="h2">
+                Enhance your <GradientText>design workflow</GradientText>
+              </Typography>
+            }
+            description="Reach out for the Design kits and the MUI Connect plug-in to bridge the gap between development and design when using Material UI."
+          />
+          <Group sx={{ mt: 4, pb: { xs: 0, md: 2 } }}>
+            <Highlighter disableBorder selected={customized} onClick={() => setCustomized(true)}>
+              <Item
+                icon={<ExtensionRoundedIcon color="primary" />}
+                title="MUI Connect"
+                description="This Figma plug-in generates a theme file with the customizations done on the Material UI design kit, allowing you to export it and paste it into your codebase."
+              />
+            </Highlighter>
+            <Highlighter disableBorder selected={!customized} onClick={() => setCustomized(false)}>
+              <Item
+                icon={<DrawRoundedIcon color="primary" />}
+                title="Design kits"
+                description="The Design kits contain many of the Material UI components with states, variations, colors, typography, and icons. We frequently update it to sync with the most up-to-date release."
+              />
+            </Highlighter>
```

Comment:
I think "Design kits" should come first. Without using this, there are no point in having the other product.